zero_bin/leader/src/main.rs

```diff
@@ -36,6 +37,10 @@ fn get_previous_proof(path: Option<PathBuf>) -> Result<Option<GeneratedBlockProo
     Ok(Some(proof))
 }
 
+const SEGMENT_PROOF_ROUTING_KEY: &str = "segment_proof";
```

Comment:
Are we using these whole strings to make debugging easier?

Reply:
These are routing key/queue names. Didnt understand exact point